heuristic/repair_operators.py

```diff
@@ -641,7 +664,7 @@ def worst_employee_regret_repair(competencies, t_covered_by_shift, employee_with
         above_maximum_demand(state, destroy_set, employee_with_competencies, demand, competencies, t_covered_by_shift)
         below_minimum_demand(state, destroy_set, employee_with_competencies, demand, competencies, t_covered_by_shift)
 
-        if any(state.hard_vars["below_minimum_demand"]):
+        if sum(state.hard_vars["below_minimum_demand"].values()) != 0:
```

Comment:
Why do you change this? The correct way should be to use:
```python
any(state.hard_vars["below_minimum_demand"])
```

Reply:
It didn't seem to work as expected when i tested it.

Reply:
<img width="296" alt="bilde" src="https://user-images.githubusercontent.com/1371159/82642896-ed201780-9c0e-11ea-819e-7744f3bdb550.png">


```python
any(dict)
```
looks for entries in the dict, while
```python
any(dict.values())
```
looks for non-zero values in the dict

Reply:
Would need .values(), then